package.json

```diff
@@ -24,7 +24,8 @@
     "lint:styles": "stylelint 'src/**/*.js'",
     "parity": "parity --config parity-config.toml",
     "precommit": "lint-staged",
-    "prep": "mkdir build/ && mkdir build/embed && cp src/index.html build/ && mkdir build/assets/ && cp -r src/assets/* build/assets/",
+    "prep": "mkdir build/ && mkdir build/embed && cp src/views/index.hbs build/ && mkdir build/assets/ && cp -r src/assets/* build/assets/ && yarn run prep:clappr",
+    "prep:clappr": "cp -r node_modules/clappr/dist/* build/",
```

Comment:
This is no longer on `dev` and should be removed!

Reply:
you mean the clappr script?